style.css

```diff
@@ -0,0 +1,156 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* navbar section styling */
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 12px;
+  left: 0%;
+  right: 0%;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  position: absolute;
+  flex-wrap: wrap;
+  margin-right: 100px;
+}
+
+.navbar a {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 10px;
+  font-size: 19px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 10px;
+  text-decoration: none;
+}
+
+.navbar nav {
+  align-self: flex-end;
+}
+
+.navbar img {
+  display: flex;
+  right: 27.67%;
+  left: 16%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid #fff;
+  margin-bottom: 3px;
+  flex-grow: 0;
+  order: 0;
+  flex-direction: row;
+}
+
+#bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-self: flex-end;
+  margin: 2px 0 2px 10px;
+}
+
+#bar img {
+  display: flex;
+  right: 27%;
+  left: 16.67%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid white;
+  margin-bottom: 3px;
+}
+
+/* headline section styling */
+.headline {
+  overflow: hidden;
+  background-image: url(images/Group.png);
+  background-position-y: 125%;
+  background-color: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  text-align: left;
+  height: auto;
+  width: 100%;
+  padding: 160px 25px;
+  gap: 15px;
+  background-position: right bottom;
+  background-size: 145px;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.h1 {
+  color: #ff6b00;
+  width: 500px;
+  height: 5px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 210;
+  font-size: 44px;
+  line-height: 65px;
+  letter-spacing: 0.35px;
+  flex: none;
+}
```

Comment:
- [ ] We should try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=pxjXectK2eF6R2jZ-0). If you notice the heading in the Figma template design covers almost 80-90% width of the screen size which is not the case with your heading that covers more than 100%. Kindly ensure to fix that to match the Figma design.
- [ ] [OPTIONAL] Kindly consider using the descriptive class selectors. This will help us to ensure that there are no other classes that inherit your stylings unknowingly ☺️.
Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/227119717-0c075b87-cbe5-49c7-b543-5f2abaa6c631.png)

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/227119980-81e65745-1139-46c9-9d77-405e3f083174.png)